UI/resolution_picker.js

```diff
@@ -11,6 +11,11 @@ class ResolutionPicker {
      */
     constructor(resolution_select_id) {
         this._selector = document.getElementById(resolution_select_id);
+        addEventListener("change", (e) => {
+            // Run UpdateResolution on the scene
+            let res = this.GetResolution();
+            Config.scene.UpdateResolution(res);
+        });
```

Comment:
Take another look at this, I don't think this code does anything.
I'll let you figure out why 😉 (ask me if you get stuck)

Reply:
Oh, I can import
```js
import Scene from "../Scene/scene.js";
```
and then
```js
addEventListener("change", (e) => {
    let res = this.GetResolution();
    Scene.UpdateResolution(res);
});
```